source/app/metrics/utils.mjs

```diff
@@ -333,13 +333,16 @@ export async function imgb64(image, {width, height, fallback = true} = {}) {
   try {
     //Fix: redirections are not properly supported by jimp (https://github.com/oliver-moran/jimp/issues/909)
     if (typeof image === "string") {
```

Comment:
I think since the bug only occurs in testing workflow which makes it fail, maybe it's better to keeep the change of cd3d526 but instead change this condition to `((typeof image === "string")&&(process.env.SANDBOX))`, so theorically the "redirection hotfix" will only happens on ci workflow, and it should work again in prod while all build passes too

Reply:
done 👍

Reply:
Seems that `JEST_WORKER_ID` may be more appropriate, will merge this if tests are passing 👍